openshift/system/Dockerfile

```diff
@@ -94,14 +91,16 @@ ENV THINKING_SPHINX_ADDRESS=0.0.0.0 \
     THINKING_SPHINX_LOG=/dev/stdout \
     RAILS_ENV=production \
     SECRET_KEY_BASE=dummy \
+    SAFETY_ASSURED=1 \
```

Comment:
Why this? are we running migrations here? We shouldn't need this to run `db:create` and `db:schema:load`. Did you face the Strong Migrations error?

Reply:
Yes, it failed without it.

Reply:
I just tried removing `SAFETY_ASSURED` from the Dockerfile and it builds the image successfully for me. And it seems logical because as I mentioned `db:create` and `db:schema:load` create the DB from the schema, which already includes all migrations. So no migration should run.

On the other hand, a couple lines above you set `RAILS_ENV=production`, but `strong_migrations` only triggers the alert on `development`.

Reply:
I'll try again, maybe I was running manually in a container and got confused when db was not empty.

Reply:
removed